README.md

```diff
@@ -49,7 +102,7 @@ import the framework with `import Kommunicate`.
 
 ## Docs
 
-Please checkout [Documentation](https://docs.kommunicate.io/docs/ios-authentication.html) for more details.
+Please check the [documentation](https://docs.kommunicate.io/docs/ios-authentication.html) for more details on installation, configuration, and customization.
```

Comment:
@parthsh url we can use install section url https://docs.kommunicate.io/docs/ios-installation
same we can check for android read me